Library/Formula/awscli.rb

```diff
@@ -69,8 +69,13 @@ class Awscli < Formula
   end
 
   resource "rsa" do
-    url "https://pypi.python.org/packages/source/r/rsa/rsa-3.1.2.tar.gz"
-    sha1 "ebf54ad3fff8bc1df09f5d777d5a913e5aef8df5"
+    url "https://pypi.python.org/packages/source/r/rsa/rsa-3.1.4.tar.gz"
+    sha1 "208583c49489b7ab415a4455eae7618b7055feca"
+  end
+
+  resource "wsgiref" do
+    url "https://pypi.python.org/packages/source/w/wsgiref/wsgiref-0.1.2.zip"
+    sha1 "80b7e9b039e40a2f8419e00b393a6516d80cf8de"
```

Comment:
Or, rather, wsgiref was added to the stdlib in 2.5 and this formula is guaranteed to be run with 2.7 so I think it's unnecessary.